sql/queries.sql

```diff
@@ -362,3 +362,236 @@ CREATE VIEW user_balance
+#vi 
+-- Report of posted jobs by an employer during a specific period of time (Job
+-- title, date posted, short description of the job up to 50 characters, number
+-- of needed employees to the post, number of applied jobs to the post,
+-- number of accepted offers).
+
+SELECT posting.title, posting.date_posted, posting.description, posting.number_of_openings,
+ COUNT(application.application_id), COUNT(application.status)
+FROM posting p, application app
+WHERE p.posting_id=app.posting_id AND p.employer_id=1 AND
+ date_posted BETWEEN '2019-01-01' AND CURDATE() AND app.status='accepted';
+
+-- -----------------------------------------------------------------------
+#vii 
+-- CREATING AN EMPLOYEE
+
+INSERT INTO users (user_id, email, password)
+    VALUES (12, 'agohier011@booking.com', 'xpnTTJ811');
+    
+INSERT INTO job_seeker (job_seeker_id, first_name, last_name, description, job_seeker_tier)
+	VALUES (12, 'Jen', 'Smith', 'receptionist', 0);
+    
+-- EDITING AN EMPLOYEE INFO
+UPDATE job_seeker
+SET description='Babysitting'
+WHERE job_seeker_id=12;
+
+-- DISPLAYING AN EMPLOYEE
+SELECT *
+FROM job_seeker
+WHERE job_seeker_id=12;
+
+-- DELETING AN EMPLOYEE
+DELETE FROM users
+WHERE user_id=12;
+
+DELETE FROM job_seeker
+WHERE job_seeker_id=12;
+
+-- -----------------------------------------------------------------------
+#viii
+-- Search for a job by an employee.
+
+INSERT INTO search(search_id, user_id, input)
+	VALUES(7, 10, 'software developer');
+
+INSERT INTO job_seeker_search(job_seeker_id, search_id)
+	VALUES(10, 7);
+    
+-- -----------------------------------------------------------------------
+#ix
+-- Apply for a job by an employee
+
+INSERT INTO application(job_seeker_id, application_id, employer_id, posting_id)
+	VALUES(8, 5, 3, 2);
+     
+ -- -----------------------------------------------------------------------
+ #X
+ -- Accept/Deny a job offer by an employee
+ 
+ UPDATE application
+ SET status='accepted'
+ WHERE job_seeker_id=6 AND application.application_id=1;
+ 
+ -- -----------------------------------------------------------------------
+#xi
+-- Withdraw from an applied job by an employee
+
+UPDATE application
+ SET status='withdrawn'
+ WHERE job_seeker_id=6 AND application.application_id=1;
+ 
+-- -----------------------------------------------------------------------
+#xii
+-- Delete a profile by an employee
+
+DELETE FROM application
+WHERE job_seeker_id=6 AND application.application_id=1;
+
+-- -----------------------------------------------------------------------
+#xiii
+
+-- Report of applied jobs by an employee during a specific period of time
+-- (Job title, date applied, short description of the job up to 50 characters,
+-- status of the application).
+
+
+SELECT posting.title, posting.date_posted, posting.description, application.status
+FROM posting p, application app
+WHERE p.posting_id=app.posting_id AND app.job_seeker_id=6 AND
+ date_posted BETWEEN '2019-01-01' AND CURDATE();
```

Comment:
should be posting_date, not date_posted, and it needs to have a timestamp part; use CURRENT_TIMESTAMP instead of CURDATE